.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,48 @@
+name: Docker build
+
+on:
+  # Disabled to not break development
+  push:
+    branches:
+      - PM-1523-build-workflow # TODO: change to `main`
+    #tags:
+      #- '*'
+  #pull_request:
+    #branches:
+      #- main
+  
+  # Only for testing
+  workflow_dispatch:
+
+env:
+  ECR_REPOSITORY_URL: 673156464838.dkr.ecr.us-west-2.amazonaws.com
+  ECR_REPOSITORY_NAME: gpt-survey-summarizer
```

Comment:
**thought:** Given the repo is public, should we move these values to secrets ? What do you think @kaozenn

Reply:
I agree. The data do not need to be exposed

Reply:
Do you have any proposal on how to do this? @kaozenn @johnmarcou 

I am looking into [github actions secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository) and I could do this manually and use the secrets from the workflow as described.
Do we want to do this actually with [infrastructure as code](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/actions_secret) rather than manually?

Reply:
I think we can keep that on the side for now 👍 

But, to answer your question, it could be done like [here](https://github.com/MinaFoundation/seeds/settings/secrets/actions) and [here](https://github.com/MinaFoundation/seeds/blob/main/.github/workflows/publish.yaml#L25).

Let's close it for now though, it was just a thought.